elisp-bytecode.texi

```diff
@@ -1133,6 +1244,7 @@ but simply call the corresponding C function.
 * bobp::
 * current-buffer::
 * set-buffer::
+* byte-save-current-buffer::
```

Comment:
I'd mildly prefer `save-current-buffer-OBSOLETE` and `save-current-buffer` as our names for these instructions.